lib/espejote.libsonnet

```diff
@@ -0,0 +1,65 @@
+/**
+ * \file espejote.libsonnet
+ * \brief Helpers to create Espejote CRs.
+ *        API reference: https://github.com/vshn/espejote/blob/main/docs/api.adoc
+ */
+
+local helper = import 'component/helper.libsonnet';
+local roles = import 'component/roles.libsonnet';
+local groupVersion = 'espejote.io/v1alpha1';
+
+/**
+  * \brief Helper to create JsonnetLibrary objects.
+  *
+  * \arg The name of the JsonnetLibrary.
+  * \arg The namespace of the ManagedResource.
+  * \return A JsonnetLibrary object.
+  */
+local jsonnetLibrary(name, namespace) = {
+  apiVersion: groupVersion,
+  kind: 'JsonnetLibrary',
+  metadata: {
+    labels: {
+      'app.kubernetes.io/name': name,
+    },
+    name: name,
+    namespace: namespace,
+  },
+};
+
+/**
+  * \brief Helper to create ManagedResource objects.
+  *
+  * \arg The name of the ManagedResource.
+  * \arg The namespace of the ManagedResource.
+  * \return A ManagedResource object.
+  */
+local managedResource(name, namespace) = {
+  apiVersion: groupVersion,
+  kind: 'ManagedResource',
+  metadata: {
+    labels: {
+      'app.kubernetes.io/name': name,
+    },
+    name: name,
+    namespace: namespace,
+  },
+};
+
+/**
+  * \brief Helper to generate RBAC objects to read resources from context and triggers.
+  *
+  * \arg The manifest of the ManagedResource.
+  * \return A list of RBAC objects.
+  */
+local generateRbacObjects(mrName) =  // TODO: Refactor to use with manifest instead of mrName
```

Comment:
Where does this function the get manifest from? 🙈

Reply:
Yeah, did you read the `TODO` 🤨

Reply:
I'm confused if the variable is just wrongly names or if it's `__some_global_state_all_mrs[mrName]`

Reply:
The argument is supposed to be the whole manifest, for now its the name of the `params.managedResources` dict. So you can't use it ouside of the component. But for that there are a chain of other functions to be updated as well.
As of now, I "just" copied the function over from `main.jsonnet` thats why it doesn't work.

Reply:
We might just remove the function for the initial release and re-add later.